### PR TITLE
Add spaceship and crappy controls.

### PR DESCRIPTION
**Pull request by TyrenDe**

### CODE REVIEW

File: client/app/game/Spaceship.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        this.player.body.maxAngular = Spaceship.MAXSPEED;
+        this.player.body.drag.setTo(Spaceship.DRAG, Spaceship.DRAG);
+        this.player.body.angularDrag = Spaceship.ANGULARDRAG;
+
+        game.camera.follow(this.player);
+
+        this.cursors = game.input.keyboard.createCursorKeys();
+    }
+
+    update(game: Phaser.Game) {
```
Comment (adamk33n3r) [inline client/app/game/Spaceship.ts]: Why does this function need this param? Doesn't it get it in the constructor?

File: client/app/routes/home/home.controller.ts [hunk 2]
```diff
@@ -2,21 +2,35 @@
...
+import Spaceship from '../../game/Spaceship';
+import Enemy from '../../game/Enemy';
 
 import Phaser = require('phaser');
 
 import angular = require('angular');
 angular.module('spacial_conquest')
-.controller('HomeController', class {
-    test: string;
+.controller('HomeController', class HomeController {
```
Comment (adamk33n3r) [inline client/app/routes/home/home.controller.ts]: What's the point of this change?

Comment (TyrenDe) [inline client/app/routes/home/home.controller.ts]: Oh, originally, I had added some static fields and I needed to access them. I don't need it now.

File: client/app/game/Spaceship.ts (hunk 1 above)
Comment (TyrenDe) [inline client/app/game/Spaceship.ts]: Yeah, I depending on my usage, I didn't want every object to hold a reference to the Phaser.Game.  *shrug*  I will probably refactor that away in a future change.

File: client/app/routes/home/home.controller.ts (hunk 2 above)
Comment (adamk33n3r) [inline client/app/routes/home/home.controller.ts]: Well having a name literally does nothing for us here. But that brings up a point. I think that this should be moved out of the angular controller. This should really just be handling angular/page stuff

Comment (TyrenDe) [inline client/app/routes/home/home.controller.ts]: Yeah, I had started refactoring stuff out (Spaceship.ts is the start of this).  I'll move more out as I can.

**Pull request merged**